website/content/docs/getting-started.md

```diff
@@ -175,6 +196,59 @@ The included example topologies:
 * `TaskHookTopology.java` --- A topology that uses a task hook to subscribe to
    event notifications.
 
+## Frequently Asked Questions
```

Comment:
Agree. Another question is: should we keep one single troubleshooting page for the whole heron, or keep two: one for topology developers while another for heron developers (or operators).